hash/oric1_cass.xml

```diff
@@ -1849,6 +2006,30 @@ Type-in from magazine
 		</part>
 	</software>
 
+	<software name="impomiss">
+		<description>Impossible Mission (Atmos 50Hz)</description>
+		<year>2010</year>
+		<publisher>Defence Force</publisher>
+		<info name="usage" value="only Oric Atmos compatible" />
+		<part name="cass" interface="oric1_cass">
+			<dataarea name="cass" size="48110">
+				<rom name="im10.tap" size="48110" crc="0a02a6de" sha1="abfc6eee380488c8548001b17f1b6e2716f430ac"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="impomiss_a" cloneof="impomiss">
+		<description>Impossible Mission (Atmos 60Hz)</description>
+		<year>2010</year>
+		<publisher>Defence Force</publisher>
+		<info name="usage" value="only Oric Atmos compatible" />
+		<part name="cass" interface="oric1_cass">
+			<dataarea name="cass" size="48110">
+				<rom name="im10-60.tap" size="48110" crc="225c6d82" sha1="09009d75d67e7e7501561028898559a68280820e"/>
+			</dataarea>
+		</part>
+	</software>
```

Comment:
Wouldn’t compatibility be handled better with compatibility tags?

Reply:
You're right!